src/component/utils/DraftFeatureFlags-core.js

```diff
@@ -6,13 +6,14 @@
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
  *
- * @providesModule DraftFeatureFlags
+ * @providesModule DraftFeatureFlags-core
  * @flow
  */
 
 'use strict';
 
 var DraftFeatureFlags = {
+  draft_accept_selection_after_refocus: false,
```

Comment:
Do you know what this is for?

Reply:
This flag was used in a Facebook internal version of Draft. It is added here because the file was brought in sync with Facebook's internal version.